src/store/lokapi.ts

```diff
@@ -49,8 +49,10 @@ export var moduleLokAPI = {
       await commit("setBalCurr")
       await commit("setThisWeekTransactions")
     },
-    async genPaymentLink({commit}:any,amount:number) {
-      await commit("genPaymentLink", amount)
+    async genPaymentLink({ commit, state }:any, amount:number) {
+      console.log(state.accounts)
+      // state.paymentUrl = await state.accounts[0].getCreditUrl(amount)
+      // console.log("paymentUrl url =", state.paymentUrl.order_url)
```

Comment:
Quite sure that we broke something here on this commit ;-) (`state.paymentUrl` is not set)

Reply:
We broke behaviour... that was already broken, but at least we don't trigger unnecessary creditUrls.
And state.paymentUrl is an empty string right from the start so no harm.
But I guess you don't like it because it is not fixing data fetching upon app refresh.